mafiasi/base/templates/base.html

```diff
@@ -21,6 +21,9 @@
     {% endblock %}
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
     <link rel="stylesheet" href="{% static 'css/darkmode.css' %}" />
+    <script>
```

Comment:
This render blocking script-tag is required to prevent the page from flashing in a wrong theme